Wrappers.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+#include "Car.h"
+#include "Studio.h"
+#include "Plane.h"
+#include "Fleet.h"
+#include "MegaStudio.h"
```

Comment:
Don't include what you don't use.
None of these types are being used in this header file.
"used" means that member variables and member functions are being used.
These types are being used in the cpp file.
Forward declare the types here.
include the appropriate header files in the cpp file.
- [ ] revise